src/App/App.jsx

```diff
@@ -138,11 +133,6 @@ function App() {
     setSelectedItem(item);
   };
 
```

Comment:
Please, reuse the `closeActiveModal()` function for any closing, to avoid `setActiveModal("")` code duplication